charts/apisix-ingress-controller/templates/servicemonitor.yaml

```diff
@@ -30,8 +30,9 @@ metadata:
   {{- end }}
 spec:
   endpoints:
-  - targetPort: http
+  - targetPort: prometheus
```

Comment:
The original HTTP should be the metrics of ingress. 
You should add servicemonitor instead of replacing it.

Why replace it with the metrics of the data plane? https://github.com/apache/apisix-helm-chart/blob/master/charts/apisix/templates/service-monitor.yaml#L41

How did you deploy it?

Reply:
Yeah, you're right. I will rewrite the PR. To scrape both apisix-ingress-controller and apisix (gateway) properly.